test/integration_test.py

```diff
@@ -148,11 +148,21 @@ def test(self):
         if config.deployment_stage != 'prod':
             try:
                 self._test_indexing()
+            except Exception as e:
+                log.error('Exception potentially due to timeout expiring while'
+                          'waiting for queues to drain. Cleaning up ...', exc_info=e)
```

Comment:
Speculating like this will only cause confusion in someone reading the logs.

```suggestion
                log.error('Exception while indexing. Cleaning up ...', exc_info=e)
```

Reply:
I don't see how this branch will end up with the test failing. Is the swallowing of the exception intentional? A mere log message is almost always the wrong way to handle fatal errors.

Reply:
Yea, non intentional. I remember you saying that it is non-practical to catch exceptions and re-raised them, or perhaps that isn't the way to go. I missed this one.